package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@pinia/testing": "^0.0.14",
-    "@sigrennesmetropole/cooperation_jn_common_ui": "github:sigrennesmetropole/cooperation_jn_common_ui#87a63a0",
+    "@sigrennesmetropole/cooperation_jn_common_ui": "github:sigrennesmetropole/cooperation_jn_common_ui#718ef47",
```

Comment:
I think we should use the same approach as in `trambus`, pin the version by the package version. But we need this to be merged first: https://github.com/sigrennesmetropole/cooperation_jn_solar/pull/17/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L33

Reply:
Yes, the change will be made in this other PR :)